libs/nuxt/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@nx-plus/nuxt",
+  "version": "0.0.1",
+  "main": "src/index.js",
+  "schematics": "./collection.json",
+  "builders": "./builders.json",
+  "description": "Nuxt plugin for Nx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ZachJW34/nx-plus.git"
+  },
+  "keywords": [
+    "nx",
+    "nuxt",
+    "vuex",
+    "jest",
+    "cypress",
+    "monorepo"
+  ],
+  "author": "Bucky Maler <maler.bucky@gmail.com> (http://buckymaler.com/)",
+  "contributors": [
+    "Zachary Williams <zachjw34@gmail.com> (https://github.com/ZachJW34/)"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/ZachJW34/nx-plus/issues"
+  },
+  "homepage": "https://github.com/ZachJW34/nx-plus/tree/master/libs/nuxt#readme",
+  "peerDependencies": {
+    "@nrwl/workspace": "^9.0.0"
+  },
+  "dependencies": {
```

Comment:
It's not listed here because it's added by the schematic. It's listed in the root `package.json` though.

Reply:
I thought we were adding the dependency if we directly use it in our source files via `import` or `require`. I'm good both ways, just want to verify.

Reply:
In this case, that's why it's added to the root `package.json`. It would also be added to the library `package.json` if it wasn't installed by the schematic, but there's no reason for it to be there since it's installed by the schematic.